docs/specs/restore.yml

```diff
@@ -16,7 +16,7 @@ inputs:
     [!include[include to 2](~/docfx-test-dependencies2/docs/a.md)]
 outputs:
   .errors.log: |
-    ["warning","link-out-of-scope","File 'docs/a.md [docfx-test-dependencies1]' referenced by link '~/docfx-test-dependencies1/docs/a.md' will not be built because it is not included in build scope","docs/a.md",1,1]
+    ["warning","link-out-of-scope","File 'docfx-test-dependencies1/docs/a.md [docfx-test-dependencies1]' referenced by link '~/docfx-test-dependencies1/docs/a.md' will not be built because it is not included in build scope","docs/a.md",1,1]
```

Comment:
do we still need `[docfx-test-dependencies1]` since we have it in the file path now?